### ISSUE TITLE
I can download videos, but their video size is small and they cannot be played.

### ISSUE THREAD

**Issue by IwanPWK** [ReneR97/skillshare-downloader#6]
please fix it
From the beginning the subtitles couldn't be downloaded, ok, I didn't have a problem, but the download process always stopped in the middle of the process, I repeated the download process many times but it always stopped in the middle of the process, finally I got around it by changing

"let downloaded and for loop"

The download process is smooth, but there is a new problem, the size of some videos is small and cannot be played

sorry for my bad English, I hope you understand what I mean

the link https://www.skillshare.com/en/classes/The-Complete-React-Bootcamp-2023-Updated/895676015/projects?via=search-layout-grid

**Comment by IwanPWK** [comment]
![evidence](https://github.com/ReneR97/skillshare-downloader/assets/130124669/f512dde6-d6ce-4b57-a205-ee1640a3501b)

**Comment by ReneR97** [comment]
Hello, 

did you use the latest version of the tool? 
There was a problem with the older version when there where to many sections in a single course

**Comment by IwanPWK** [comment]
Yes, I'm using the latest version of the tool, the problem only occurs in some recent courses or updated courses
